backend/documents/models.py

```diff
@@ -0,0 +1,25 @@
+import uuid
+from django.db import models
+
+# Create your models here.
+from course_evaluations.models import EOCSpecific, EOCGeneral
+
+class Document(models.Model):
```

Comment:
document needs to be aligned to a particular `courseEvaluation` right? 

It might be useful as well to modify the `course_evaluations/serializers.py` to include the documents as well that a courseEvaluation has

Reply:
done